### PR TITLE
feat: add `GetUserByUserId`

### PR DESCRIPTION
**Pull request by qianxi0410**
Close: #25

### CODE REVIEW

File: casdoorsdk/user.go [hunk 1]
```diff
@@ -188,6 +188,27 @@ func GetUserByEmail(email string) (*User, error) {
...
 
+func GetUserByUserId(userId string) (*User, error) {
+	queryMap := map[string]string{
+		"owner":  authConfig.OrganizationName,
+		"userId": userId,
+	}
+
+	url := GetUrl("get-user", queryMap)
+
+	bytes, err := DoGetBytesRaw(url)
```
Comment (hsluoyz) [inline casdoorsdk/user.go:199]: should it be `DoGetBytes()`?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: casdoorsdk/user.go (hunk 1 above)
Comment (qianxi0410) [reply casdoorsdk/user.go:199]: In fact, in my tests, only when using `DoGetBytesRaw`, can the test succeed.

图片

图片

And note that when using email for inquiries, you should also use `DoGetBytesRaw`.

**Comment by qianxi0410** [comment]
PTAL @hsluoyz

File: casdoorsdk/user.go (hunk 1 above)
Comment (hsluoyz) [reply casdoorsdk/user.go:199]: Yes, both should use `DoGetBytesRaw()`, no problem now.

**Pull request merged**